### PR TITLE
feat(hero): add `hero` component examples

### LINKED ISSUE #83: Add `Hero headers`
### Subject

Hero headers

### Description

Add [Hero headers](https://ui.mantine.dev/category/hero/)

### Are you willing to participate in the development of this feature and create a pull request?

None

### PR DESCRIPTION
**Pull request by taroj1205**
Closes #83

## Description

I have added a few hero headers examples.

## Is this a breaking change (Yes/No):

No

### CODE REVIEW

File: contents/page-sections/hero-headers/with-background-image/index.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  return (
+    <Container
+      bgImage="url(https://images.unsplash.com/photo-1573164713988-8665fc963095?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=980&q=80)"
+      bgSize="cover"
+      bgPosition="center"
+      minH="fit-content"
+      h="full"
+    >
+      <Box position="absolute" inset={0} zIndex="0" bg="blackAlpha.700" />
+      <VStack as={Center} px="xl" h="full" zIndex="1" py="xl" gap="lg">
```
Comment (illionillion) [inline contents/page-sections/hero-headers/with-background-image/index.tsx:23]: If we specify `px` and `py` as the same thing, why not just use `p`?
And I think you should change the padding only for `md` and `sm`.

```suggestion
      <VStack as={Center} h="full" zIndex="1" p={{base: "lg", md: "md"}} gap="lg">
```

File: contents/page-sections/hero-headers/with-content-on-left/index.tsx [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+  return (
+    <Container
+      bgImage="url(https://images.unsplash.com/photo-1519389950473-47ba0277781c?ixlib=rb-1.2.1&ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&auto=format&fit=crop&w=1770&q=80)"
+      bgSize="cover"
+      bgPosition="center"
+      minH="fit-content"
+      h="full"
+    >
+      <Box position="absolute" inset={0} zIndex="0" bg="blackAlpha.700" />
+      <VStack as={Center} px="xl" h="full" zIndex="1" py="xl" gap="lg">
```
Comment (illionillion) [inline contents/page-sections/hero-headers/with-content-on-left/index.tsx:22]: Buttons have been crushed.

image

**Review by illionillion — CHANGES_REQUESTED** [review]
@taroj1205 

Thanks again for making another great component.
I would like to see a few modifications.

File: contents/page-sections/hero-headers/with-content-on-left/index.tsx (hunk 2 above)
Comment (taroj1205) [reply contents/page-sections/hero-headers/with-content-on-left/index.tsx:22]: The button does not seem to be crushed on my end. Do you know the cause?
image

File: contents/page-sections/hero-headers/with-background-image/index.tsx (hunk 1 above)
Comment (taroj1205) [reply contents/page-sections/hero-headers/with-background-image/index.tsx]: Fixed.

**Review by illionillion — APPROVED** [review]
LGTM

**Review by hirotomoyamada — APPROVED** [review]

**Pull request merged**